messages/src/lib.rs

```diff
@@ -267,12 +267,13 @@ pub enum HwError {
     )]
     WaitFailed,
 
-    /// The FPGA reported an I2C error
+    /// The FPGA reported an I2C error.  u32 is a logical mask for which
+    /// ports saw a failure.
     #[cfg_attr(
         any(test, feature = "std"),
-        error("FPGA reported an I2C error, module may not be present")
+        error("FPGA reported an I2C error, module may not be present. Logical mask for which ports saw a failure: 0x{0:08x}")
```

Comment:
I would personally love to see this error report the actual indices of the modules which reported failure. The value in the `I2cError` enum variant can and should still be the bit-mask type, but the human-level error that we print, for example in `xcvradm` could be just the list of indices.

[Here](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=00fa5467289f99e35e70d51d5d86e711) is a playground link that shows how you'd do this. `thiserror` is extremely flexible, and we can use the `PortMask::to_indices` method to turn the bitmask into a list of indices in the error message printed. Hopefully that makes sense, and let me know if that seems reasonable!

Reply:
I'll dig into `thiserror` more and get it to print out each individual index. That seems like a very reasonable and helpful thing to do!